plugins/Observability/src/Observability.cpp

```diff
@@ -44,10 +44,10 @@
 #include "StelObject.hpp"
 #include "StelObjectMgr.hpp"
 #include "StelObserver.hpp"
```

Comment:
My QtCreator suggests that following headers may be removed:
StarMgr, StelFader, StelFileMgr, StelIniParser, StelObserver, StelSkyDrawer, ZoneArray.

It is possible that omission of one header requires inclusion of a "lighter" one. Just make sure it compiles.

Reply:
I'll remove the unnecessary ones and see if it compiles.